openfl/federated/data/federated_data.py

```diff
@@ -74,15 +81,18 @@ def __init__(self, X_train, y_train, X_valid, y_valid,
         self.valid_splitter = self._get_splitter_or_default(valid_splitter)
 
     @staticmethod
-    def _get_splitter_or_default(value):
+    def _get_splitter_or_default(
+        value: Optional[NumPyDataSplitter]
+    ) -> Union[NumPyDataSplitter, EqualNumPyDataSplitter]:
```

Comment:
```suggestion
    ) -> NumPyDataSplitter:
```
`EqualNumPyDataSplitter` is a subclass of `NumPyDataSplitter`.